frontend/pages/portfolios.vue

```diff
@@ -0,0 +1,170 @@
+      size="small"
+    >
+      <PVColumn selection-mode="multiple" />
+      <PVColumn
+        field="editorObject.value.createdAt.originalValue"
+        header="Created At"
+        sortable
+      >
+        <template #body="slotProps">
+          {{ humanReadableTimeFromStandardString(slotProps.data.editorObject.value.createdAt.originalValue).value }}
+        </template>
+      </PVColumn>
+      <PVColumn
+        field="editorObject.value.name.originalValue"
+        sortable
+        header="Name"
+      />
+      <PVColumn
+        expander
+        header="Details"
+      />
+      <template
+        #expansion="slotProps"
+      >
+        <div class="surface-100 p-3">
+          <h2 class="mt-0">
+            Metadata
+          </h2>
+          <div class="flex flex-column gap-2 w-fit">
+            <div class="flex gap-2 justify-content-between">
+              <span>Created At</span>
+              <b>{{ humanReadableTimeFromStandardString(slotProps.data.editorObject.value.createdAt.originalValue).value }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Number of Rows</span>
+              <b>{{ slotProps.data.editorObject.value.numberOfRows.originalValue }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Holdings Date</span>
+              <b>{{ humanReadableDateFromStandardString(slotProps.data.editorObject.value.holdingsDate.originalValue.time).value }}</b>
+            </div>
+          </div>
+          <h2 class="mt-5">
+            Editable Properties
+          </h2>
+          <PortfolioEditor v-model:editor-portfolio="slotProps.data.editorObject.value" />
+          <div class="flex gap-3 justify-content-between">
+            <PVButton
+              icon="pi pi-trash"
+              class="p-button-danger p-button-outlined"
+              label="Delete"
+              @click="() => deletePortfolio(slotProps.data.id)"
+            />
+            <div v-tooltip.bottom="slotProps.data.saveTooltip">
+              <PVButton
+                :disabled="!slotProps.data.canSave.value"
+                label="Save Changes"
+                icon="pi pi-save"
+                icon-pos="right"
+                @click="() => saveChanges(slotProps.data.id)"
+              />
+            </div>
+          </div>
+          <StandardDebug
+            :value="slotProps.data.editorObject.value"
+            label="Portfolio Upload"
+          />
+        </div>
+      </template>
+    </PVDataTable>
+    <div class="flex flex-wrap gap-3 w-full justify-content-between">
+      <LinkButton
+        class="p-button-outlined"
+        icon="pi pi-arrow-left"
+        to="/upload"
+        label="Upload New Portfolios"
+      />
+      <PVButton
+        class="p-button-outlined"
+        label="How To Run a Report"
+        icon="pi pi-question-circle"
+        icon-pos="right"
+      />
```

Comment:
I'd add a TODO or something to hook this up to something, just so we don't forget about it